src/select-list-view.js

```diff
@@ -37,6 +37,14 @@ module.exports = class SelectListView {
     this.refs.queryEditor.setText('')
   }
 
+  setQueryText (value, isSelect = false) {
```

Comment:
That's fair, although I would argue that the second parameter in `selectIndex` is an implementation detail and not really part of the API, so I would still prefer using an object if possible.

I like naming the parameter `selectText` as opposed to `selectAll`. 👍